<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TodoList</class>
 <widget class="QWidget" name="TodoList">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1286</width>
    <height>715</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_32">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1311</width>
     <height>741</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../image/WhatsApp Image 2024-11-10 Ã  18.51.31_e66bd550.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QListWidget" name="taskList">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>180</y>
     <width>301</width>
     <height>461</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
    border: 2px solid rgb(228, 196, 255);  
    border-radius: 10px; /* Coins arrondis */
    padding: 2px;
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(233, 222, 255, 255), stop:1 rgba(255, 255, 255, 255));
</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="taskInput">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>361</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">	background-color: rgb(255, 202, 213);
    border: 2px solid #EF8EBA; /* Bordure rose */
    border-radius: 10px; /* Coins arrondis */
    padding: 2px;
    </string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dueDateEdit">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>290</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">	background-color: rgb(255, 202, 213);
    border: 2px solid #EF8EBA; /* Bordure rose */
    border-radius: 10px; /* Coins arrondis */
    padding: 2px;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addButton">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>340</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background-color: rgb(255, 202, 213);
    color: white; /* Texte blanc */
    border-radius:20px  ; /* Coins arrondis */
	border-color: rgb(239, 142, 186);
    padding: 8px;
}</string>
   </property>
   <property name="text">
    <string>ADD</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteButton">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>660</y>
     <width>321</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	border: 2px solid rgb(228, 196, 255);  
    border-radius: 10px; /* Coins arrondis */
    padding: 2px;
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(233, 222, 255, 255), stop:1 rgba(255, 255, 255, 255));

}</string>
   </property>
   <property name="text">
    <string>DELETE</string>
   </property>
  </widget>
  <widget class="QPushButton" name="markDoneButton">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>660</y>
     <width>321</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	border: 2px solid rgb(228, 196, 255);  
    border-radius: 10px; /* Coins arrondis */
    padding: 2px;
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(233, 222, 255, 255), stop:1 rgba(255, 255, 255, 255));

}</string>
   </property>
   <property name="text">
    <string>MARK</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_23">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>70</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
	color: rgb(191, 139, 255);
	background-color: transparent;
	text-decoration: underline;
	font: 700 italic 20pt &quot;Segoe UI&quot;;
	;
}</string>
   </property>
   <property name="text">
    <string>TO DO LIST</string>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QListWidget" name="completedTaskList">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>180</y>
     <width>301</width>
     <height>461</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
    border: 2px solid rgb(228, 196, 255);  
    border-radius: 10px; /* Coins arrondis */
    padding: 2px;
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(233, 222, 255, 255), stop:1 rgba(255, 255, 255, 255));
</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_24">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>120</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
	color: rgb(191, 139, 255);
	background-color: transparent;
	text-decoration: underline;
	font: 700 italic 20pt &quot;Segoe UI&quot;;
	;
}</string>
   </property>
   <property name="text">
    <string>TO DO </string>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_25">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>120</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
	color: rgb(191, 139, 255);
	background-color: transparent;
	text-decoration: underline;
	font: 700 italic 20pt &quot;Segoe UI&quot;;
	;
}</string>
   </property>
   <property name="text">
    <string>DONE</string>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
